BedrockCommand.cpp

```diff
@@ -202,21 +205,24 @@ void BedrockCommand::finalizeTimingInfo() {
         }
     }
 
-    // Log all this info.
     SINFO("command '" << methodName << "' timing info (ms): "
-          << peekTotal/1000 << " (" << peekCount << "), "
-          << processTotal/1000 << " (" << processCount << "), "
-          << commitWorkerTotal/1000 << ", "
-          << commitSyncTotal/1000 << ", "
-          << queueWorkerTotal/1000 << ", "
-          << queueSyncTotal/1000 << ", "
-          << totalTime/1000 << ", "
-          << unaccountedTime/1000 << ", "
-          << escalationTimeUS/1000 << ". Upstream: "
-          << upstreamPeekTime/1000 << ", "
-          << upstreamProcessTime/1000 << ", "
-          << upstreamTotalTime/1000 << ", "
-          << upstreamUnaccountedTime/1000 << "."
+          "peek:" << peekTotal/1000 << " (count:" << peekCount << "), "
```

Comment:
OH GOD THANK YOU! I always needed to open this line to know what data was each of the numbers. I even already remember some by heart 😞